llm_guard/input_scanners/anonymize.py

```diff
@@ -94,12 +97,19 @@ def __init__(
         self._preamble = preamble
         self._use_faker = use_faker
         self._threshold = threshold
+        self._language = language
```

Comment:
Do you have validation of supported languages?

Reply:
I verified the Chinese, and the English is already supported by package-self.
But other languages I have not verified, sorry, I don't know other languages.
So I don't know what can I do next, this feature is too big for me alone.😩

Reply:
Sorry, I meant if somebody provide `language=[de]`, will it fail validation?

Reply:
Hummm, I'm missing this case. Let me try.